src/xrpld/ledger/detail/View.cpp

```diff
@@ -2838,11 +2838,12 @@ rippleUnlockEscrowMPT(
     ApplyView& view,
     AccountID const& sender,
     AccountID const& receiver,
-    STAmount const& amount,
+    STAmount const& netAmount,
+    STAmount const& grossAmount,
     beast::Journal j)
 {
-    auto const issuer = amount.getIssuer();
-    auto const mptIssue = amount.get<MPTIssue>();
+    auto const issuer = grossAmount.getIssuer();
```

Comment:
could possibly add a safety check that  if `netAmount != grossAmount && !view.enabled(fixEscrowTokenV1)` is true then return internal error